### PR TITLE
Add chapter on Open Scholarship and section on Open Education to Communication guide 

### LINKED ISSUE #2362: Chapter or section on Open Education
### Discussion Points

- [ ] What guide is best suited: Communication chapter or Reproducible research section under Open Research? 
- [ ] What is Open Education?
- [ ] Why Open Education?

## Who can help?

* Anyone with an interest in Open Education or no experience at all so you can check if the chapter makes any sense!

## Updates

- [ ] Write chapter outline: https://hackmd.io/zsi7LVSVS3yuPdXva6jhQA
- [ ] Add material to the chapter
- [ ] Combine materials into a readable chapter
- [ ] Proofread
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to main branch.

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #2362
Adding more contextual information around Open Education, other than just Open Educational Resources, and moving this Chapter to the Communication guide as this is not necessarily part of reproducible research. 

### List of changes proposed in this PR (pull-request)

* Add a chapter on Open Scholarship in the Communication guide to move the information about Open Education/Open Educational Resources there. As a follow up the Open Research chapter should probably be moved under Open Scholarship.

### What should a reviewer concentrate their feedback on?

- [ ] Do the proposed changes make sense?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/communication/open.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+
+(cm-open-prerequisites)=
+## Prerequisites
+
+No previous knowledge is needed. 
+Related information to Open Scholarship can be found in the {ref}`rr-open` chapter, that also has a more detailed summary of the importance and motivation behind Open Scholarship.
```
Comment (Arielle-Bennett) [inline book/website/communication/open.md:18]: Added in ref to Open Research chapter in reproducible guide

File: book/website/communication/open/education.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+(cm-open-education)=
+# Open Education 
+
+(cm-open-education-what)=
+## What is Open Education
+Open Education is a collective term that encompasses educational resources, tools and practices that can be freely used in the digital environment without legal, financial or technical barriers. 
+Open Education can also include [Open Pedagogy](http://openpedagogy.org/open-pedagogy/) which also focuses on the conversation with each other to develop educational practices and structures.
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:7]: Would re-phrase "on the conversation with each other to develop educational practices and structures." to "which also focuses on the process of collaboratively developing educational practices and structures"

File: book/website/communication/open/education.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+In open educational practices there is more space for [participation of learners in the knowledge creation and sharing](https://opencontent.org/blog/archives/2975), empowering learners and stimulating peer-learning and colllaboration.  
+
+(cm-open-education-why)=
+## Why Open Education?
+
+Open Education is important as it:
+
+- Facilitates broader and more equal access to access to knowledge and knowledge generation.
+Open Education is rooted in the human right to access high-quality education.
+- Is more easy to use:
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:17]: "Easier to use"

File: book/website/communication/open/education.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Open Education is important as it:
+
+- Facilitates broader and more equal access to access to knowledge and knowledge generation.
+Open Education is rooted in the human right to access high-quality education.
+- Is more easy to use: 
+    - Easier to build upon and correct the materials, which facilitates customisation of your education curriculum.
+    - Easier to collaborate, participate and share resources.
+- Reduces duplication of effort: If an educator needs materials for teaching and such materials already exist openly, then they need not make their own from scratch, thereby saving time.
+- Is generally more accessible due to open formats compared to proprietary formats in which files cannot be changed.
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:21]: "due to open" -> "due to the use of open"

File: book/website/communication/open/education.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+- Reduces duplication of effort: If an educator needs materials for teaching and such materials already exist openly, then they need not make their own from scratch, thereby saving time.
+- Is generally more accessible due to open formats compared to proprietary formats in which files cannot be changed. 
+    - Formats like Microsoft docx requires access to Microsoft Word, whereas open formats such as RTF and HTML work with a variety of software. 
+    - While .pdf is an open format, it is also more closed in the sense that it is difficult to edit compared to formats such as RTF and HTML. 
+    - Accessibility is not guaranteed! 
+Learn more about how to improve the accessibility of your resources: 
+        - [How to Create Engaging Open Educational Resources without Sacrificing Accessibility](https://www.youtube.com/watch?v=UtOCI5TqtQI)
+        - [OER and Accessibility: Working Toward Inclusive Learning](https://sparcopen.org/news/2018/oer-accessibility-working-toward-inclusive-learning/)
+- Open [does not necessairly exclude revenue generation](https://edscoop.com/how-traditional-textbook-publishers-can-do-well-by-the-oer-community/)
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:28]: "necessarily" - typo

File: book/website/communication/open/education.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+# Open Educational Resources (OERs)
+
+Open educational resources (OERs) [{term}`def<Open Educational Resources>`] are teaching and learning materials that can be freely used and reused for learning or teaching at no cost, and without needing to ask permission. Examples are courses, including Massive Online Open Courses (MOOCs), lectures, teaching materials, assignments, and various other resources. OERs are available in many different formats compatible with online usage, most obviously text, images, audio, and video. Anyone with internet access can access and use OERs; access is not dependent on location or membership of a particular institution.
+
+Unlike copyrighted resources, OERs have been authored or created by an individual or organization that chooses to retain few, if any, ownership rights. In some cases, that means anyone can download a resource and share it with colleagues and students. In other cases, this may go further and enable people to edit resources and then re-post them as a remixed work. How do you know your options? OERs often have a Creative Commons licence or other permission to let you know how the material may be used, reused, adapted, and shared.
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:48]: Link to Creative Commons license website?

File: book/website/communication/open/education.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+- _Reuse_: the right to use the content in a wide range of ways (for example, in a class, in a study group, on a website, in a video).
+- _Revise_: the right to adapt, adjust, modify, or alter the content itself (for example, translate the content into another language).
+- _Remix_: the right to combine the original or revised content with other open content to create something new (for example, incorporate the content into a mashup).
+- _Redistribute_: the right to share copies of the original content, your revisions, or your remixes with others (for example, give a copy of the content to a friend).
+
+(cm-open-education-oers-how)=
+## How to set up OERs? 
+
+1. Choose a platform.
+2. Collect resources under an open license (see [Attribution & Fair Use: Copyright in Open Education](https://www.youtube.com/watch?v=jGTUHdadqJU) and {ref} section on data licenses<rr-licensing-data>.
```
Comment (Arielle-Bennett) [inline book/website/communication/open/education.md:62]: This ref is broken, this it should be section on {ref}`rr-licensing-data`

**Review by Arielle-Bennett — APPROVED** [review]
Minor comments and typoes - one ref is broken but once it's all fixed you should be good to merge!

File: book/website/communication/open/education.md (hunk 3 above)
Comment (EstherPlomp) [reply book/website/communication/open/education.md:17]: Haha. My brain sometimes. Thanks!

File: book/website/communication/open/education.md (hunk 2 above)
Comment (EstherPlomp) [reply book/website/communication/open/education.md:7]: Changed, thanks!

File: book/website/communication/open/education.md (hunk 4 above)
Comment (EstherPlomp) [reply book/website/communication/open/education.md:21]: thank you!

File: book/website/communication/open/education.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+## Why Open Education?
+
+Open Education is important as it:
+
+- Facilitates broader and more equal access to access to knowledge and knowledge generation.
+Open Education is rooted in the human right to access high-quality education.
+- Is more easy to use: 
+    - Easier to build upon and correct the materials, which facilitates customisation of your education curriculum.
+    - Easier to collaborate, participate and share resources.
+- Reduces duplication of effort: If an educator needs materials for teaching and such materials already exist openly, then they need not make their own from scratch, thereby saving time.
```
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply book/website/communication/open/education.md:20]: Good point! Very much agree :)

File: book/website/communication/open/education.md (hunk 6 above)
Comment (EstherPlomp) [reply book/website/communication/open/education.md:48]: Great idea, thanks!

**Pull request merged**